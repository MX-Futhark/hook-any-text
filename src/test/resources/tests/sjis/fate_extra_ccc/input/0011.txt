82DC82E982C592EA82C882B582CC235255425382B182AD82C782A423525542458D9593B42352454E4482CC82E682A4816381630A82A282A282A681418CC48B7A82C982A082A682AE8B9B82CC82E682A482C8816381630A816381638B9B82CC8163816382E682A482C881480000000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000001100000012000000130000001400000015000000160000001700000018000000190000001A0000001B0000001C0000001D0000001E0000001F00000020000000210000002200000023000000240000002500000026000000270000002800000B000000E814C90800000000000000000000000000007E00000000000000F0430000A842FFFFFFFF000000000000000000000000FFFFFFFF0B000000FFFFFFFF0000504300000000000000000300100000000000FFFFFFFF0000000000008843000060C100001F4300000000FFFFFFFF000050430000884300000A4300001F430B0000006815C90800000000000000000000000007007E00000000000000184300001843FFFFFFFF000000000000000000000000FFFFFFFF0B000000FFFFFFFF0000504300000000000000000400100000010000FFFFFFFF0000000000008843000000C000002A4300000000FFFFFFFF00005043000088430000FC4200002A4302000000000000000000000000000000000000001000C20100000000C00EA3480B0000000000000000000000000000000000000002001000000000000000000000000000FFFFFFDC0000000000003C43000000000000F04300000000FFFFFFDC0000F04300003C4300000000000000000000A842FFFFFFDC0000000000008843000000000000F0430000A842FFFFFFDC0000F04300008843000000000000C0420B0000000000000000000000000000000000000003001000000000000000000000000000FFFFFFDC000060C100001F43000000000000184300000000FFFFFFDC