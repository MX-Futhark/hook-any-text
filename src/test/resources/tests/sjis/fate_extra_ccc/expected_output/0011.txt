["まるで底なしの黒洞のよう……
いいえ、呼吸にあえぐ魚のような……
……魚の……ような？"]